src/main/java/org/llorllale/cactoos/matchers/InputHasContent.java

```diff
@@ -58,7 +58,9 @@ public InputHasContent(final String text) {
     public InputHasContent(final Text text) {
         this(
             new MatcherOf<>(
-                (String input) -> text.asString().equals(input), text
+                (String input) -> text.asString().equals(input),
+                    desc -> desc.appendText(text.asString()),
+                    (actual, desc) -> desc.appendValue(actual)
```

Comment:
@ruca905 can you add some context in both the two functions in correcpondnce with this matcher responsibility? You can see other usages of `MatcherOf` as examples. `HasSize` is a good example :)